/gauge_reader:
  gauge_reader:
    ros__parameters:
      qos_overrides:
        /gauge_reading:
          publisher:
            reliability: reliable
            history: keep_last
            depth: 5
        /detections:
          subscription:
            reliability: reliable
            history: keep_last
            depth: 5
      image_stream:
        reliability: best_effort
        history: keep_last
        depth: 1

